pipeline/src/convert_csv.ts

```diff
@@ -0,0 +1,77 @@
+import { program } from "commander";
+import util from "util";
+
+import FS from 'node:fs';
+import assert from 'node:assert';
+import {parse} from 'csv-parse/sync';
+
+import fs = require("node:fs/promises");
+import path = require("node:path");
+
+import { Table, filterTables, mergeTablesByModelNumber } from "./table_merger";
+import { APPLIANCE_TYPES } from "../../backend/schema/metadata";
+import { glob } from "glob";
+import { retrieveMetadata } from "./metadata";
+
+const SPECS_FILE_BASE = "../data/";
+const INPUT_SUBDIR = "reformatted/";
+const OUTPUT_SUBDIR = "";
+const RUNS = "runs/";
+
+program.parse();
+
+async function main() {
+// loop through the sub folders in the run folder
+const rfolder = path.join(SPECS_FILE_BASE, RUNS).replace(/\\/g, '/');
```

Comment:
Instead of converting all of the record files in all of the runs every time, it might be simpler to simply have the user specify the filepath of the csv they want to convert. Then we're not tied to any particular directory structure. I think that will be more maintainable in the long run.

The `pipeline/src/generate_metadata_files.ts` script is an example of a script that takes a single input file as an argument.

Reply:
Changed to require file path like generate_metadata_files.ts